components/expenses/ExpenseFormPayeeSignUpStep.js

```diff
@@ -314,7 +314,13 @@ const ExpenseFormPayeeSignUpStep = ({ formik, collective, onCancel, onNext }) =>
           <Box>
             <Field name={payeeType === PAYEE_TYPE.ORG ? 'payee.organization.legalName' : 'payee.legalName'}>
               {({ field }) => (
-                <StyledInputField name={field.name} label={formatMessage(msg.legalName)} labelFontSize="13px" mt={3}>
+                <StyledInputField
+                  name={field.name}
+                  isPrivate
```

Comment:
It seems that we're not requiring it. Should we?

```suggestion
                  isPrivate
                  required={false}
```

Reply:
Ah yes, this is not required. I've corrected it. 👍🏽